src/main.rs

```diff
@@ -142,6 +150,12 @@ fn main() {
             .expect("'start-time-cpu_us' parameter expected to be of 'u64' type.")
     });
 
+    let vmm_config_json = if let Some(path) = cmd_arguments.value_of("config-file") {
```

Comment:
I would change this to something like this:
```
let vmm_config_json = cmd_arguments.value_of("config-file")
    .map(fs::read_to_string)
    .map(|x| x.expect("Unable to open or read from the configuration file")
```

Reply:
Done